src/node_url.cc

```diff
@@ -786,7 +786,7 @@ inline bool ToASCII(const std::string& input, std::string* output) {
 
 void URLHost::ParseIPv6Host(const char* input, size_t length) {
   CHECK_EQ(type_, HostType::H_FAILED);
-  for (unsigned n = 0; n < 8; n++)
+  for (unsigned n = 0; n < arraysize(value_.ipv6); n++)
     value_.ipv6[n] = 0;
   uint16_t* piece_pointer = &value_.ipv6[0];
   uint16_t* const buffer_end = piece_pointer + 8;
```

Comment:
should we use it also here? Or save the result and result it here?

Reply:
good catch, updated